server/routes/surveyRoutes.js

```diff
@@ -61,13 +35,13 @@ module.exports = app => {
       title,
       subject,
       body,
-      recipients: recipients.split(',').map(email => ({ email: email.trim() })),
+      recipients: surveyService.getRecipientListFromEmailString(recipients),
       _user: req.user.id,
       dateSent: Date.now()
     });
 
-    // Great place to send an email!
-    const mailer = new Mailer(survey, surveyTemplate(survey));
+    const surveyContent = surveyTemplate(survey);
+    const mailer = new Mailer(survey, surveyContent);
```

Comment:
Passing a function call into a function is kind of ugly, I'd rather have one more line of code and have it be easier to understand. So I create a new variable `surveyContent` and store the filled survey there. Then I pass it into the Mailer constructor.